lib/orderbook/OrderBook.ts

```diff
@@ -25,10 +25,14 @@ class OrderBook extends EventEmitter {
   private repository: OrderBookRepository;
   private matchesProcessor: MatchesProcessor = new MatchesProcessor();
 
-  private ownOrders: { [ pairId: string ]: Orders } = {};
-  private peerOrders: { [ pairId: string ]: Orders } = {};
+  private ownOrders: Map<String, Orders> = new Map<String, Orders>();
+  private peerOrders: Map<String, Orders> = new Map<String, Orders>();
 
-  private localIds: { [ localId: string ]: string } = {};
+  /**
+   * This map has the localId of and order as key and the global id as value
+   * which makes possible to cancel orders by their local id without iterating
```

Comment:
Minor typos:

"localId of **and** order"
"which makes **it** possible"

And I think I'd also change "to cancel orders" to "to reference orders" since we might use this for more than just canceling down the road, and I'd drop "without iterating" at the end and add a period.